e2e_samples/parking_sensors_synapse/.devcontainer/devcontainer.json

```diff
@@ -3,15 +3,20 @@
 	"dockerComposeFile": "docker-compose.yml",
 	"workspaceFolder": "/workspace/e2e_samples/parking_sensors_synapse",
 	"service": "parking_sensor_synapse_dc",
-	"extensions": [
-		"ms-python.python",
-		"eamodio.gitlens",
-		"DavidAnson.vscode-markdownlint",
-		"yzhang.markdown-all-in-one",
-		"ms-azuretools.vscode-bicep"
-	],
-	"settings": {
-		"python.pythonPath": "/usr/local/bin/python"
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"ms-python.python",
+				"eamodio.gitlens",
+				"DavidAnson.vscode-markdownlint",
+				"yzhang.markdown-all-in-one",
+				"ms-azuretools.vscode-bicep"
+			]
+		},
+		"settings": {
+			"python.pythonPath": "/usr/local/bin/python"
+		}
 	},
+	"remoteUser": "mdwuser",
 	"postCreateCommand": "./.devcontainer/post_create.sh"
```

Comment:
While testing and I get to the post_create.sh command I get the following error: 
postCreateCommandfailed with exit code 127. Skipping any further user-provided commands. 
Did you also have this problem? Maybe is just a problem on my side, but I would like to confirm with you.